src/app/state/pool/pool.effects.ts

```diff
@@ -319,4 +327,12 @@ export class PoolEffects {
         })
       );
   }
+
+  private openNotEnrolledRoleDialog() {
+    this.dialog.open(NotEnroledRoleInfoComponent, {
```

Comment:
Does the `NotEnroledRoleInfoComponent` have further logic to either redirect you to the role request or to prompt you to add to the `ClaimManager`?

Reply:
This is how it looks like: ![image](https://user-images.githubusercontent.com/83639710/144583841-66ebeebe-7c2e-40b1-be57-9d0112485d3f.png)
And here is task description: https://energyweb.atlassian.net/browse/DS-100